utils/create-component/templates/react/stories.mdx.js

```diff
@@ -0,0 +1,35 @@
+module.exports = (modulePath, storyPath, pascalCaseName, titleCaseName, prerelease, description) => `
+import {Meta} from '@storybook/addon-docs';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {${pascalCaseName}} from '${modulePath}';
+import {Basic} from './examples/Basic';
+import {Open} from './examples/Open';
+
+<Meta title="${storyPath}" component={${pascalCaseName}} />
+
+# ${titleCaseName}
+
+${description}
+
+## Installation
+
+\`\`\`sh
+yarn add @workday/canvas-kit-${prerelease && prerelease + '-'}react
+\`\`\`
+
+
+## Usage
+
+### Basic
```

Comment:
Thank you for updating this script to generate a starter MDX! Small nit to keep things consistent with our [standard MDX structure](https://github.com/Workday/canvas-kit/discussions/1131):

```suggestion
### Basic Example
```

Would love to build on this in a future PR to create a more complete starter document.

Reply:
Sure thing. Totally into the idea of building on this. IMO it is one of the best ways to learn how to create really good components. Sometimes it is hard to figure out which component you should model after.

Reply:
Would love to add cypress to the mix too